framework/audio/internal/eqinsert.h

```diff
@@ -16,39 +16,32 @@
 //  along with this program; if not, write to the Free Software
 //  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 //=============================================================================
+#ifndef MU_AUDIO_EQINSERT_H
+#define MU_AUDIO_EQINSERT_H
 
-#ifndef MU_AUDIO_LOOPSTREAM_H
-#define MU_AUDIO_LOOPSTREAM_H
-
-#include <memory>
-
-#include "iaudiosource.h"
-#include "audiotypes.h"
+#include <iaudioinsert.h>
 
 namespace mu {
 namespace audio {
-class LoopSource : public IAudioSource
+//! Example of eq insert. It used fixed biquad equation for 1 band eq
+class EqInsert : public IAudioInsert
```

Comment:
it is better not to use abbreviations in names

Reply:
@Eism file and class renamed `class Equaliser : public IAudioInsert` Thanks